sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTest.java

```diff
@@ -124,11 +121,6 @@
 public class TextAnalyticsClientTest extends TextAnalyticsClientTestBase {
     private TextAnalyticsClient client;
 
-    @BeforeAll
-    static void beforeAll() throws InterruptedException {
-        TimeUnit.SECONDS.sleep(180);
```

Comment:
We don't have sleeps like this anywhere else do we? If there are other cases like this we should remove them if possible.

Reply:
> We don't have sleeps like this anywhere else do we? If there are other cases like this we should remove them if possible.

Yes. It is the only place. I remember it is because resource creation used to take time to be available and it is a temporary fix but probably forget to remove it.